website/static/js/addProjectPlugin.js

```diff
@@ -86,12 +86,15 @@ var AddProject = {
             self.isValid(false);
         };
     },
-    view : function (ctrl, options) {
+    view : function (ctrl, options, trackingCategory, trackingAction) {
```

Comment:
This is counter to the usual signature of component `view` methods, which is `view(ctrl, options, children)` (where `children` is an optional array of mithril elements, as in `m.component(MyPanel, {size: 'large'}, [m('span',{}, 'Hello world')])`.

Better to make `trackingCategory` and `trackingAction` part of the `options` object. This makes for better readability when using the component. Compare

```js
// What do 'foo' and 'bar' mean?
m.component(AddProject, {someParam: 42}, 'foo', 'bar')
```

vs.

```js
m.component(AddProject, {someParam: 42, trackingCategory: 'foo', trackingAction: 'bar'})
```